swaglyrics/tab.py

```diff
@@ -1,21 +1,42 @@
 from swaglyrics.cli import lyrics
-from flask import Flask, render_template
+from flask import Flask, render_template,jsonify,request
+from flask_cors import CORS, cross_origin
 import os
 from swaglyrics import spotify
 
+
 app = Flask(__name__, template_folder=os.path.join(os.path.dirname(os.path.abspath('SwagLyrics-For-Spotify')), 'swaglyrics/templates'))
-# use relative path of the template folder
+
+CORS(app, support_credentials=True)
+
 
 song = None
 artist = None
+isChrome = False
+chromeSong=None
+isSameSong=True
 
-
-@app.route('/')
+@app.route('/',methods=["GET","POST"])
+@cross_origin(supports_credentials=True)
 def tab():
     # format lyrics for the browser tab template
-    global song, artist
-    song = spotify.song()
-    artist = spotify.artist()
+    global song, artist,isChrome,isSameSong
```

Comment:
spacing is uneven

Reply:
@aadibajpai Add the @pep8speaks integration and this won't happen anymore ;)

Reply:
@canihavesomecoffee It's there in the repo. I think subsequent changes should work